.env.development

```diff
@@ -0,0 +1,4 @@
+PORT=8080
+REACT_APP_API_PATH_PREFIX='/v1'
+REACT_APP_API_URL=http://localhost:3000
+REACT_APP_ENVIRONMENT=production
```

Comment:
Empty line at the end of the file for all .env files

Reply:
ok